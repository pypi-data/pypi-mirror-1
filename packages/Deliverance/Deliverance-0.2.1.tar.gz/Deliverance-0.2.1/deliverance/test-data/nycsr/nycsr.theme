
<!-- helper file for doing hand transforms with this theme --> 

<blend 
   theme="http://www.nycsr.org" 
   baseurl="http://www.nycsr.org" 
   rules="./test-data/nycsr/nycsr.xml" /> 
