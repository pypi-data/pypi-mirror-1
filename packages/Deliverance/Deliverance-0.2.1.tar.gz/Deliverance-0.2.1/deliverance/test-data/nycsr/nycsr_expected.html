<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<style type="text/css"><!-- @import url(http://www.openplans.org/portal_css/Plone%20Default/ploneStyles3910.css); --></style>
<style type="text/css"><!-- @import url(http://www.openplans.org/portal_css/Plone%20Default/ploneStyles7546.css); --></style>
<style type="text/css"><!-- @import url(http://www.openplans.org/portal_css/Plone%20Default/ploneStyles3831.css); --></style>
<link rel="alternate stylesheet" type="text/css" media="screen" href="http://www.openplans.org/portal_css/Plone%20Default/ploneStyles3993.css" title="Small Text">
<link rel="alternate stylesheet" type="text/css" media="screen" href="http://www.openplans.org/portal_css/Plone%20Default/ploneStyles5069.css" title="Large Text">
<link rel="alternate stylesheet" type="text/css" media="screen" href="http://www.openplans.org/portal_css/Plone%20Default/ploneStyles3656.css" title="Small Text">
<link rel="alternate stylesheet" type="text/css" media="screen" href="http://www.openplans.org/portal_css/Plone%20Default/ploneStyles8929.css" title="Large Text">
<link rel="shortcut icon" type="image/x-icon" href="http://www.openplans.org/favicon.ico">
<link rel="home" href="http://www.openplans.org" title="Front page">
<link rel="search" title="Search this site" href="http://www.openplans.org/search_form">
<link rel="author" title="Author information" href="http://www.openplans.org/author/Jacqueline">
<link rel="contents" href="http://www.openplans.org/sitemap" title="Site Map">
<link rel="up" href="http://www.openplans.org/projects" title="Up one level">
<meta name="author" content="New York City Streets Renaissance">
<meta name="description" content="">
<meta name="keywords" content="">
<meta name="generator" content="BBEdit 8.2">
<meta name="MSSmartTagsPreventParsing" content="true">
<meta name="robots" content="all">
<style type="text/css" media="screen">@import url(http://codespeak.net/css/style.css);</style>
<style type="text/css" media="screen">@import url(http://codespeak.net/css/nav.css);</style>
<script type="text/javascript"><!--//--><![CDATA[//><!--
		startList = function() {
			if (document.all&&document.getElementById) {
				navRoot = document.getElementById("menu");
				for (i=0; i<navRoot.childNodes.length; i++) {
					node = navRoot.childNodes[i];
					if (node.nodeName=="LI") {
						node.onmouseover=function() {
							this.className+=" over";
						}
						node.onmouseout=function() {
							this.className=this.className.replace(" over", "");
						}
					}
				}
			}
		}
		window.onload=startList;
		
		//--><!]]>
	</script><script type="text/javascript" src="http://www.openplans.org/projects/nycstreets/openplans.js"></script><script type="text/javascript" src="http://www.openplans.org/portal_javascripts/Plone%20Default/ploneScripts6565.js">
                </script><script type="text/javascript" src="http://www.openplans.org/portal_javascripts/Plone%20Default/ploneScripts0569.js">
                </script><meta name="generator" content="Plone - http://plone.org">
<meta http-equiv="imagetoolbar" content="no">
<title>
        NYCSR 
        â€”
        OpenPlans
    </title>
</head>
<body id="page-nyc" class="gallery">
<!-- Header --><div id="header"><h1 class="logo"><a href="http://codespeak.net/"><span>The New York City Streets Renaissance</span></a></h1></div>
<!-- / Header --><hr>
<script type="text/javascript"><!--//--><![CDATA[//><!--
startList = function() {
	if (document.all&&document.getElementById) {
		navRoot = document.getElementById("menu");
		for (i=0; i<navRoot.childNodes.length; i++) {
			node = navRoot.childNodes[i];
			if (node.nodeName=="LI") {
				node.onmouseover=function() {
					this.className+=" over";
				}
				node.onmouseout=function() {
					this.className=this.className.replace(" over", "");
				}
			}
		}
	}
}
window.onload=startList;

//--><!]]></script><!-- Top Level Links --><div id="toplevel"><ul class="links">
<li id="top-home"><a href="http://codespeak.net/"><strong><span>Home Page</span></strong></a></li>
<li id="top-join"><a href="http://codespeak.net/involved/join.php"><strong><span>Join Us</span></strong></a></li>
</ul></div>
<!-- / Top Level Links --><!-- Navigation Bar --><div id="nav">
<ul id="menu" class="tabs">
<li id="nav-who">
<a href="http://codespeak.net/who/"><strong><span>Who We Are</span></strong></a><ul class="sub">
<li><a href="http://codespeak.net/who/supporters.php"><span>Supporters List</span></a></li>
<li><a href="http://codespeak.net/who/contact.php"><span>Contact Us</span></a></li>
</ul>
</li>
<li id="nav-nyc" class="selected">
<a href="http://codespeak.net/nyc/"><strong><span>New York Streets</span></strong></a><ul class="sub">
<li><a href="http://codespeak.net/nyc/truth.php"><span>The Truth About Traffic</span></a></li>
<li><a href="http://codespeak.net/nyc/broken.php"><span>Broken Streets</span></a></li>
<li><a href="http://codespeak.net/nyc/building.php"><span>Building on Progress</span></a></li>
<li><a href="http://codespeak.net/nyc/video.php"><span>StreetFilms Video Gallery</span></a></li>
<li><a href="http://codespeak.net/nyc/photo.php"><span>Photo Gallery</span></a></li>
</ul>
</li>
<li id="nav-lessons">
<a href="http://codespeak.net/lessons/"><strong><span>Global Lessons</span></strong></a><ul class="sub">
<!--
					<li>
						<a href="/lessons/contested.php">
							<span>Contested Streets</span>
						</a>
					</li>
					--><li><a href="http://codespeak.net/lessons/chicago.php"><span>Chicago</span></a></li>
<li><a href="http://codespeak.net/lessons/philadelphia.php"><span>Philadelphia</span></a></li>
<li><a href="http://codespeak.net/lessons/london.php"><span>London</span></a></li>
<li><a href="http://codespeak.net/lessons/paris.php"><span>Paris</span></a></li>
<li><a href="http://codespeak.net/lessons/copenhagen.php"><span>Copenhagen</span></a></li>
<li><a href="http://codespeak.net/lessons/great.php"><span>What Makes a Great Street?</span></a></li>
<li><a href="http://codespeak.net/lessons/benefits.php"><span>The Benefits of Great Streets</span></a></li>
<li><a href="http://codespeak.net/lessons/photo.php"><span>Photo Gallery</span></a></li>
</ul>
</li>
<li id="nav-involved">
<a href="http://codespeak.net/involved/"><strong><span>Get Involved</span></strong></a><ul class="sub">
<li><a href="http://codespeak.net/involved/join.php"><span>Join Us</span></a></li>
<li><a href="http://codespeak.net/involved/ways.php"><span>Ways to Get Involved</span></a></li>
<li><a href="http://codespeak.net/involved/resources.php"><span>Resources</span></a></li>
</ul>
</li>
<li id="nav-events"><a href="http://codespeak.net/events/"><strong><span>Events</span></strong></a></li>
</ul>
<div class="clear"></div>
</div>
<!-- / Navigation Bar --><hr>
<!-- Container --><div id="container"><table id="portal-columns"><tbody><tr>
<td id="portal-column-content">

                    
                      <div id="content" class="documentEditable">

                        
                          

        <h5 class="hiddenStructure">Views</h5>

        <ul class="contentViews">
<li id="contentview-view" class="selected">
                <a href="http://www.openplans.org/projects/nycstreets/project-home">View</a>
              </li>

            
            
              <li id="contentview-edit" class="plain">
                <a href="http://www.openplans.org/projects/nycstreets/project-home/edit">Edit</a>
              </li>
            

        </ul>
<div class="contentActions">

        

        

        





    </div>
                        

                        <div class="documentContent" id="region-content">

                          <a name="documentContent"></a>

                          

                          

                          
    

        <div class="documentActions">
    

    <h5 class="hiddenStructure">Document Actions</h5>

    <ul>
<li>
            <a href="http://www.openplans.org/projects/nycstreets/project-home/sendto_form">

                <img src="http://www.openplans.org/mail_icon.gif" alt="Send this page to somebody" title="Send this page to somebody"></a>
        </li>
    
    
        <li>
            <a href="javascript:this.print();">

                <img src="http://www.openplans.org/print_icon.gif" alt="Print this page" title="Print this page"></a>
        </li>

    

    </ul>
</div>

        <h1 class="documentFirstHeading">NYCSR </h1>
    
        <div>

    <div class="documentByLine">
        

        

        <span>
        last modified
        </span>

        2006-09-15 11:18

        

        

        

        

        <div class="reviewHistory">
    
</div>

    </div>

</div>

        
    

    
        <div class="plain">
            <h2 class="heading">
<b>Welcome to the NYC Streets Renaissance Campaign Headquarters</b><br>
</h2>
<h3 class="subheading">The goal of the NYCSR is to tranform New York City streets so that they are safer, more productive and more livable. </h3>
<p><img class="image-inline" src="http://www.openplans.org/projects/nycstreets/Sociable%20Men-1.jpg" height="200" width="314"><b><em>Streets can be places for neighbors to meet</em></b></p>
<p>New York City is defined by its vibrant and diverse streets and neighborhoods. Unfortunately, our neighborhoods and business districts are buckling under increasing amounts of dangerous car and truck  traffic.  Children can no longer 
  
     
      
	<span class="wicked_resolved" id="text-860339cd275a35aed13ed4d80e8a8ff6">

	  <a href="http://www.openplans.org/projects/nycstreets/play-on-their-own-blocks">
            <img src="http://www.openplans.org/projects/nycstreets/document_icon.gif"><span class="chunk">play on their own blocks</span>
          </a>
	</span>
      
    
    
  
  

, while parents worry about turning cars smashing into baby carriages. Senior citizens are losing their independence, shut up in their homes for fear of crossing the street.   And shoppers and investors are being turned away by chaotic, traffic-choked avenues.  </p>
<p>This is unacceptable. <strong>The time is long overdue for our great city to strike a better balance between traffic and the needs of pedestrians.   </strong></p>
<p>The <strong>NYC Streets Renaissance Campaign</strong> aims to: </p>
<ul type="disc">
<li>
<b>Educate</b> New Yorkers about potential transportation policy changes that will improve quality of life across New York City</li>
<li>
<b>Promote</b> a rebalancing of this public space away from private vehicles and toward community needs</li>
<li>
<b>Demonstrate</b> the widespread public support for reform on these issues</li>
<li>
<b>Tap </b>the potential of New Yorkers to re-imagine their own streets</li>
</ul>
<p><strong></strong></p>
<p><strong>Our Initiatives:</strong></p>
<ul type="none">
<li>Official launch with an <a title="MAS Exhibition Preparations" href="../mas/project-home" target="_self">exhibition</a>/event at the Municipal Arts Society</li>
<li>Preparation of a multimedia information packet, including a <a title="inclusionletter" href="inclusionletter" target="_self">inclusion letter</a>, a dvd featuring <a title="TOPP: video production" href="../topp-video-production/project-home" target="_self">Clarence's work</a>, and a intro booklet</li>
<li>Improvements <a title="website building" href="website-building" target="_self">requested</a> for the <a href="http://www.nycsr.org/" target="_self">nycstreets.org </a>homepage</li>
<li>Host many <a title="Postcard Revised" href="../mas/mas-nycsr-postcard-revision-2.pdf" target="_self">events</a> to supplement our exhibition at the MAS.</li>
<li>Promote and distribute 
  
     
      
	<span class="wicked_resolved" id="text-401890cd2f8c74f7d8106ecbddde78fe">

	  <a href="http://www.openplans.org/projects/nycstreets/studies-and-papers">
            <img src="http://www.openplans.org/projects/nycstreets/document_icon.gif"><span class="chunk">studies and papers</span>
          </a>
	</span>
      
    
    
  
  

 <a href="../studies-and-papers"></a>that dispel myths of streets and transit in NYC.</li>
</ul>
<p></p>
<p></p>
<ul type="none"><li>Help neighborhood leaders in the fight for long overdue, common sense improvements to their neighborhood streets</li></ul>
<p></p>
<p><strong></strong></p>
<p><strong>To get involved</strong>, add your ideas for making New York City a better place to live and work to our Taking it Citywide 
  
     
      
	<span class="wicked_resolved" id="text-215018586e8c7cf5386a1bdcdae59e1a">
	  <a href="http://www.openplans.org/projects/nycstreets/neighborhoods">

            <img src="http://www.openplans.org/projects/nycstreets/document_icon.gif"><span class="chunk">Neighborhoods</span>
          </a>
	</span>
      
    
    
  
  

 page, or visit us at <a href="http://www.nycsr.org/involved.php">http://www.nycsr.org/involved.php</a> </p>
<p></p>
<p></p>
<p></p>
<p></p>
<p><a title="old landing page" href="old-landing-page" target="_self">old landing page </a><br></p>
<p><br></p>
<p><strong>Together, we can re-imagine the streets of New York City.</strong></p>


<!-- <p>Grand Street Before:<br /><img class="image-inline" src="http://www.openplans.org/projects/nycstreets/Grand_street_Before-1.jpg" height="237" width="501" /><br /></p><p></p><p>Grand Street Re-imaged as public space:</p><p><img class="image-inline" src="http://www.openplans.org/projects/nycstreets/Grand_street_AFTER-1.jpg" height="336" width="800" /></p> --> 
        </div>

        <div class="visualClear"><!-- --></div>

        

	
	

        
        <div id="relatedItems">
    
        
            
        
    
</div>
    
    


                          
                            <div class="discussion">
    
        
            
            

        
    
</div>
                          

                        </div>

                      </div>

                    
                  </td>
                  

                  
                  
                  
                </tr></tbody></table></div>
<!-- / Container --><hr>
<!-- Footer --><div id="footer">
<p class="links">
		[ 
		<a href="http://codespeak.net/"><span>Home</span></a> 
		| 
		<a href="http://codespeak.net/who/"><span>Who We Are</span></a> 
		| 
		<a href="http://codespeak.net/nyc/"><span>New York Lessons</span></a> 
		| 
		<a href="http://codespeak.net/lessons/"><span>Global Lessons</span></a> 
		| 
		<a href="http://codespeak.net/involved/"><span>Get Involved</span></a> 
		| 
		<a href="http://codespeak.net/events/"><span>Events</span></a> 
		| 
		<a href="http://codespeak.net/involved/join.php"><span>Join Us</span></a> 
		]
		</p>
<p class="cc"><a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/2.5/"><img alt="Creative Commons License" border="0" src="http://codespeak.net/img/logos/cc.png" class="logo"><br>
				These works are licensed under a Creative Commons License
			</a></p>
<!--
		<rdf:RDF xmlns="http://web.resource.org/cc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
			<Work rdf:about="">
				<license rdf:resource="http://creativecommons.org/licenses/by-nc-nd/2.5/" />
				<dc:type rdf:resource="http://purl.org/dc/dcmitype/Text" />
			</Work>
			<License rdf:about="http://creativecommons.org/licenses/by-nc-nd/2.5/">
				<permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
				<permits rdf:resource="http://web.resource.org/cc/Distribution"/>
				<requires rdf:resource="http://web.resource.org/cc/Notice"/>
				<requires rdf:resource="http://web.resource.org/cc/Attribution"/>
				<prohibits rdf:resource="http://web.resource.org/cc/CommercialUse"/>
			</License>
		</rdf:RDF>
		-->
</div>
<!-- / Footer -->
</body>
</html>

